src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -429,6 +453,36 @@ def _enable_module_qat(self, module: Module):
             prepare_embeddings_qat(module, reduce_range=self._reduce_range)
         self._qat_enabled = True
 
+        if self._calibration_dataloader is not None:
+            self._calibrate(module)
+
+    def _calibrate(self, module):
+        if not self._calibration_dataloader or not self._qat_enabled:
+            return
+
+        _LOGGER.info("Running quantization calibration using calibration_dataloader")
+
+        module_training = module.training
+        module.eval()
+
+        forward_fn: Callable = (
+            self._calibration_function
+            if self._calibration_function
+            else partial(tensors_module_forward, module=module)
+        )
+
+        model_device = next(module.parameters()).device
+
+        for batch in self._calibration_dataloader:
+            batch = tensors_to_device(batch, model_device)
+            with torch.no_grad():
+                try:
+                    forward_fn(batch)
+                except Exception:
+                    module(batch)
```

Comment:
let's remove the try catch and let the exception occur. especially if a custom function is used, this may be hard to debug

Reply:
Agreed fixed!